WordPressAuthenticator/NUX/Button/NUXStackedButtonsViewController.swift

```diff
@@ -0,0 +1,227 @@
+        }
+    }
+
+    func configureDivider() {
+        guard showDivider else {
+            return dividerStackView.isHidden = true
+        }
+        let color = WordPressAuthenticator.shared.unifiedStyle?.borderColor ?? WordPressAuthenticator.shared.style.primaryNormalBorderColor
+        leadingDividerLine.backgroundColor = color
+        leadingDividerLineHeight.constant = WPStyleGuide.hairlineBorderWidth
+        trailingDividerLine.backgroundColor = color
+        trailingDividerLineHeight.constant = WPStyleGuide.hairlineBorderWidth
+        dividerLabel.textColor = color
+        dividerLabel.text = NSLocalizedString("Or", comment: "Divider on initial auth view separating auth options.").localizedUppercase
+    }
+
+    func updateShadowViewEdgeConstraints() {
+        guard let layoutGuide = shadowLayoutGuide,
+              let shadowView = shadowView else {
+            return
+        }
+
+        NSLayoutConstraint.deactivate(shadowViewEdgeConstraints)
+        shadowView.translatesAutoresizingMaskIntoConstraints = false
+
+        shadowViewEdgeConstraints = [
+            layoutGuide.leadingAnchor.constraint(equalTo: shadowView.leadingAnchor),
+            layoutGuide.trailingAnchor.constraint(equalTo: shadowView.trailingAnchor),
+        ]
+
+        NSLayoutConstraint.activate(shadowViewEdgeConstraints)
+    }
+
+    // MARK: - Dynamic type
+    func didChangePreferredContentSize() {
+        reloadViews()
+    }
+}
+
+extension NUXStackedButtonsViewController {
+    override open func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        if previousTraitCollection?.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory {
+            didChangePreferredContentSize()
+        }
+    }
+}
+
+extension NUXStackedButtonsViewController {
+
+    /// Sets the parentViewControlleras the receiver instance's container. Plus: the containerView will also get the receiver's
+    /// view, attached to it's edges. This is effectively analog to using an Embed Segue with the NUXButtonViewController.
+    ///
+    public func move(to parentViewController: UIViewController, into containerView: UIView) {
+        containerView.translatesAutoresizingMaskIntoConstraints = false
+        containerView.addSubview(view)
+        containerView.pinSubviewToAllEdges(view)
+
+        willMove(toParent: parentViewController)
+        parentViewController.addChild(self)
+        didMove(toParent: parentViewController)
+    }
+
+    /// Returns a new NUXButtonViewController Instance
+    ///
+    public class func instance() -> NUXStackedButtonsViewController {
+        let storyboard = UIStoryboard(name: "NUXButtonView", bundle: WordPressAuthenticator.bundle)
+        guard let buttonViewController = storyboard.instantiateViewController(withIdentifier: "NUXStackedButtonsViewController") as? NUXStackedButtonsViewController else {
```

Comment:
Nit: we can use `UIStoryboard+Helpers` to make this simpler and not have to keep the identifiers here in case there's a failure due to a typo.

Reply:
Done in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/695/commits/ad24260bc933ecb94f5aee32763ccd081a541916 and https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/695/commits/a0e0c975ff70ec52bf10d2c900058d501c2f7672